.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @vincent-chpd
+* @lucagi89
```

Comment:
Why did you add your username? lol

Reply:
Cause I saw this file and I tought is cool if we add everyone's username

Reply:
No that's the point ! lol I should be the only code owner. Please remove your name and push your branch again 😄